### PR TITLE
Add CSL style for the journal Developmental Dynamics

### PR DESCRIPTION
**Pull request by enozkan**
Developmental Dynamics, a society journal published since 1901, lacks a dedicated style. It is based on AMA, but with small modifications (include DOI as https and include publisher-place for books) according to published papers and examples given on the journal's webpage: https://https://anatomypubs.onlinelibrary.wiley.com/hub/journal/10970177/homepage/forauthors.html#refs

Zotero Forum discussion here: https://forums.zotero.org/discussion/18413/style-request-developmental-dynamics#latest

### CODE REVIEW

**Comment by enozkan** [comment]
The file passes the validator for both 1.0.1 and 1.0.2 schema, and the csl file works as expected in Zotero. Not sure how to fix it.

File: developmental-dynamics.csl [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    <title-short>Dev Dyn</title-short>
+    <id>http://www.zotero.org/styles/developmental-dynamics</id>
+    <link href="http://www.zotero.org/styles/developmental-dynamics" rel="self"/>
+    <link href="http://www.zotero.org/styles/american-medical-association" rel="template"/>
+    <link href="https://https://anatomypubs.onlinelibrary.wiley.com/hub/journal/10970177/homepage/forauthors.html#refs" rel="documentation"/>
+    <author>
+      <name>Engin Ozkan</name>
+      <email>enozkan78@gmail.com</email>
+    </author>
+    <contributor>
```
Comment (POBrien333) [inline developmental-dynamics.csl:14]: remove all the contributors. You're the author and the creators of AMA are acknowledged via the template link.

**Comment by enozkan** [comment]
Still fails with a locale-related error. Help appreciated.

**Comment by bwiernik** [comment]
Thanks!

**Comment by adam3smith** [comment]
@bwiernik -- please don't merge PRs with failing CI -- since the CI runs on all styles, this breaks CI testing for all subsequent PRs. If there is a problem with Sheldon, we need to fix it before merging.

Here, though, Sheldon actually worked correctly, spotting an invalid documentation URL (the https is duplicated) and the only problem is the unparseable error message.

**Comment by bwiernik** [comment]
Sorry my bad.

**Comment by enozkan** [comment]
Thank you for identifying my duplicated "https". I could not see it, and the error message did not help.

**Pull request merged**